Updated community talk series list

emrekiciman opened the pull request:
- reversed sort order to show news talks at the top
- added Kun Zhang's upcoming talk, with a link to join live.

emrekiciman committed:
Update community-videos.md

Changing sort order on community videos

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman committed:
Create 03_talk_series.md

Added Kun Zhang's talk abstract to videos list

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman committed:
Update 03_talk_series.md

fixed colon quoting

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman committed:
Update 03_talk_series.md

Added Kun's name to title line

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman committed:
Update 03_talk_series.md

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

amit-sharma commented on community-videos.md:
```diff
@@ -3,4 +3,4 @@ layout: page
 
-{% include articles.html collection="community_videos" %}
\ No newline at end of file
+{% include articles.html collection="community_videos" sort-order="desc" %}
```
thanks for the quick turnaround. However, one of the "videos" in the list is actually the intro to the pywhy series. With this sort, it might go to the last. can you try adding that "video", 00_talk_series.md as a text inside the community-videos.md?

emrekiciman replied:
I moved that 00_talk_series to 99_talk_series so that it will show up at the top in reverse order.

It would take more time to figure out how to add the header in a correctly formatted way to the beginning of community-videos.md

emrekiciman committed:
Rename 00_talk_series.md to 99_talk_series.md

Moved the description of the talk series to '99' so that it shows up at the top of the sort order

Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

fverac commented on _community_videos/03_talk_series.md:
```diff
@@ -0,0 +1,17 @@
+  Speaker: Kun Zhang is currently on leave from Carnegie Mellon University (CMU), where he is an associate professor of philosophy and an affiliate faculty in the machine learning department; he is working as a professor and the acting chair of the machine learning department and the director of the Center for Integrative AI at Mohamed bin Zayed University of Artificial Intelligence (MBZUAI). He  ...
+  <br>
+
+  <b><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_N2E0NzAxOTctMDAxNC00ZTY2LWE5ODYtZDU5YjhmNmRlZmM4%40thread.v2/0?context=%7b%22Tid%22%3a%22492f1487-e76c-454d-aad1-28c9aaf849f3%22%2c%22Oid%22%3a%22404ab0c2-59ec-4b36-88e9-81f01946470f%22%7d">Join the live seminar</a> on January 29, 2024 at Monday 8:00am pacific / 11:00am eastern / 4:00pm GMT / 9:30pm IST.
```
``` suggestion 
  <b><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_N2E0NzAxOTctMDAxNC00ZTY2LWE5ODYtZDU5YjhmNmRlZmM4%40thread.v2/0?context=%7b%22Tid%22%3a%22492f1487-e76c-454d-aad1-28c9aaf849f3%22%2c%22Oid%22%3a%22404ab0c2-59ec-4b36-88e9-81f01946470f%22%7d">Join the live seminar</a> on January 29, 2024 at Monday 8:00am pacific / 11:00am eastern / 4:00pm GMT / 9:30pm IST.</b>
```

This needs to be added to avoid a weird display issue

emrekiciman committed:
Update _community_videos/03_talk_series.md

Co-authored-by: fverac <fabiovera@microsoft.com>
Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

fverac commented on _community_videos/03_talk_series.md:
```diff
@@ -0,0 +1,17 @@
+  <br>
+  The talk will include a description of the causal-learn package in PyWhy. Learn more: https://github.com/py-why/causal-learn
+  <br>
+  Speaker: Kun Zhang is currently on leave from Carnegie Mellon University (CMU), where he is an associate professor of philosophy and an affiliate faculty in the machine learning department; he is working as a professor and the acting chair of the machine learning department and the director of the Center for Integrative AI at Mohamed bin Zayed University of Artificial Intelligence (MBZUAI). He  ...
```
```suggestion
  <b>Speaker:</b> Kun Zhang is currently on leave from Carnegie Mellon University (CMU), where he is an associate professor of philosophy and an affiliate faculty in the machine learning department; he is working as a professor and the acting chair of the machine learning department and the director of the Center for Integrative AI at Mohamed bin Zayed University of Artificial Intelligence (MBZUAI). He develops methods for making causality transparent by torturing various kinds of data and investigates machine learning problems including transfer learning, representation learning, and reinforcement learning from a causal perspective. He has been frequently serving as a senior area chair, area chair, or senior program committee member for major conferences in machine learning or artificial intelligence, including UAI, NeurIPS, ICML, IJCAI, AISTATS, and ICLR.  He was a general & program co-chair of the first Conference on Causal Learning and Reasoning (CLeaR 2022), a program co-chair of the 38th Conference on Uncertainty in Artificial Intelligence (UAI 2022), and is a general co-chair of UAI 2023.
```
Think bolding "speaker" here adds more structure and less "wall of text"

fverac commented on _community_videos/03_talk_series.md:
```diff
@@ -0,0 +1,17 @@
+summary: >-
+  Causality is a fundamental notion in science, engineering, and even in machine learning. Causal representation learning aims to reveal the underlying high-level hidden causal variables and their relations. It can be seen as a special case of causal discovery, whose goal is to recover the underlying causal structure or causal model from observational data. The modularity property of a causal sys ...
+  <br>
+  The talk will include a description of the causal-learn package in PyWhy. Learn more: https://github.com/py-why/causal-learn
```
```suggestion
  The talk will include a description of the causal-learn package in PyWhy. Learn more: <a href="https://github.com/py-why/causal-learn">https://github.com/py-why/causal-learn</a>
```
Fixing link

emrekiciman committed:
Update _community_videos/03_talk_series.md

Co-authored-by: fverac <fabiovera@microsoft.com>
Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman committed:
Update _community_videos/03_talk_series.md

Co-authored-by: fverac <fabiovera@microsoft.com>
Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

fverac commented on _community_videos/03_talk_series.md:
```diff
@@ -0,0 +1,17 @@
+  Causality is a fundamental notion in science, engineering, and even in machine learning. Causal representation learning aims to reveal the underlying high-level hidden causal variables and their relations. It can be seen as a special case of causal discovery, whose goal is to recover the underlying causal structure or causal model from observational data. The modularity property of a causal sys ...
+  <br>
+  The talk will include a description of the causal-learn package in PyWhy. Learn more: <a href="https://github.com/py-why/causal-learn">https://github.com/py-why/causal-learn</a>
+  <br>
```
```suggestion
<br><br>
```
I subjectively think adding an additional line break here is better

fverac commented:
<img width="663" alt="image" src="https://github.com/py-why/py-why.github.io/assets/22120253/51b71b1a-3c69-447b-a20a-b5ddcdda57ea">
preview of what it should look like with my suggestions

emrekiciman committed:
Update _community_videos/03_talk_series.md

Co-authored-by: fverac <fabiovera@microsoft.com>
Signed-off-by: Emre Kıcıman <emrek@microsoft.com>

emrekiciman commented:
Thanks @fverac ! I've merged all your suggestions into the PR!

fverac reviewed (approved):
Thanks for putting the PR together!
